tests/vshn.yml

```diff
@@ -5,6 +5,9 @@ parameters:
         source: https://raw.githubusercontent.com/projectsyn/component-crossplane/v2.3.0/lib/crossplane.libsonnet
         output_path: vendor/lib/crossplane.libsonnet
 
+  facts:
+    cloud: dev
+
```

Comment:
Instead of matching on a cloud `dev` I'd add a separate kapitan compile step. Then there is no chance that this will end up in production